runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -2799,7 +2799,7 @@ MM_CopyForwardScheme::getNextWorkUnit(MM_EnvironmentVLHGC *env, UDATA preferredN
 					while (!isAnyScanWorkAvailable(env) && (doneIndex == _doneIndex)) {
 #if defined(J9MODRON_TGC_PARALLEL_STATISTICS)
 						PORT_ACCESS_FROM_ENVIRONMENT(env);
-						U_64 waitEndTime, waitStartTime;
+						uint8_t waitEndTime, waitStartTime;
```

Comment:
I think you meant `uint64_t`, and each local should be declared (and initialized separately).